.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+  "targets": ["last 2 chrome versions", "last 2 firefox versions"],
+  "plugins": [
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "regenerator": true
+      }
+    ]
+  ],
+  "presets": [
+    "@babel/preset-env",
+    ["@babel/preset-react", { "runtime": "automatic" }],
+    "@babel/preset-typescript",
+    "babel-preset-jest"
```

Comment:
This seems to be automatically added when using babel-jest (which is also automatic when running jest and babel-jest is installed), so could possibly remove this line. Also I'm a bit worried that this transform is always present even when not running tests (that said I have no idea what this does).

update: I see that `babel-jest` is not installed in this project, so if everything is running fine with this config, I suggest to handle the proposed change in a future PR.